### PR TITLE
Unassign screen WorkAllocation 2

### PR DESCRIPTION
**Pull request by kuda1992**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3596

### Change description ###

1. Mocked the node layer to return hard coded list of tasks.
2. Made changes to the unassign screen

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/work-allocation-2/components/constants copy/index.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { CONFIG_CONSTANTS as ConfigConstants } from './config.constants';
```
Comment (connorpgpmcelroy) [inline src/work-allocation-2/components/constants copy/index.ts:1]: See comment above

File: src/work-allocation-2/components/task-field/task-field.component.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.xui-image-field__container {
```
Comment (connorpgpmcelroy) [inline src/work-allocation-2/components/task-field/task-field.component.scss:1]: Again, this file should not exist. This should be present in file called task-field.component.scss

File: src/work-allocation-2/containers/task-assignment/task-assignment-container.component.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.task-assignment-btn {
```
Comment (connorpgpmcelroy) [inline src/work-allocation-2/containers/task-assignment/task-assignment-container.component.scss:1]: This styling does not affect anything and should not be added again - File should be removed from PR

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
PR is re-adding things I've previously removed because it branched off before I had pushed the changes so they need to be re-removed

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by udaydenduluri33 — APPROVED** [review]
LGTMk

**Pull request merged**